packages/graphql/tests/integration/interface-relationships/declare-relationship/interface-filters.int.test.ts

```diff
@@ -480,11 +480,11 @@ describe("interface filters of declared interface relationships", () => {
                 episodeNr,
             }
         );
-        await session.close();
     });
 
     afterEach(async () => {
-        await cleanNodes(driver, [Movie, Series, Actor, Episode]);
+        await cleanNodesUsingSession(session, [Movie, Series, Actor, Episode]);
+        await session.close();
```

Comment:
I believe that session is already closed by the `cleanNodesUsingSession`, see: [`packages/graphql/tests/utils/run-cypher.ts` line: 26](https://github.com/neo4j/graphql/blob/dev/packages/graphql/tests/utils/run-cypher.ts#L26)